packages/esm-appointments-app/src/appointments/details/appointment-details.component.tsx

```diff
@@ -42,14 +41,22 @@ const AppointmentDetails: React.FC<AppointmentDetailsProps> = ({ appointment })
             <p className={styles.labelBold}>{t('gender', 'Gender')}: </p>
             <p className={styles.label}>{getGender(appointment.patient.gender, t)}</p>
           </div>
-          <div className={styles.labelContainer}>
-            <p className={styles.labelBold}>{t('dateOfBirth', 'Date of birth')}: </p>
-            <p className={styles.label}>
-              {patientDetails && patientDetails?.dateOfBirth
-                ? formatDate(new Date(patientDetails && patientDetails?.dateOfBirth ? patientDetails?.dateOfBirth : ''))
-                : ''}
-            </p>
-          </div>
+          {patient && patient?.birthDate ? (
+            <div className={styles.labelContainer}>
+              <p className={styles.labelBold}>{t('dateOfBirth', 'Date of birth')}: </p>
+              <p className={styles.label}>{formatDate(new Date(patient.birthDate))}</p>
+            </div>
+          ) : (
+            ''
+          )}
+          {patient && patient?.telecom
+            ? patient.telecom.map((contact, i) => (
+                <div className={styles.labelContainer}>
+                  <p className={styles.labelBold}>{t('Telephone', 'Phone {{index}}', { index: i + 1 })}: </p>
```

Comment:
Although the general examples so far are for Telephone, I think we could store email addresses or other contact types here.  It looks like there's some support for getting up mappings of contact point types, but until we do that let's change this to the more generic "Contact" that would be correct for both Telephone and and email, ie :  {t('contact'. 'Contact {{index}}'